docs/guide/01-api-protobuf.md

```diff
@@ -13,78 +13,94 @@ keywords:
   - HTTP
 ---
 
-这里主要进行修订Proto规范约定和多语言之间特定商定，帮助大家写出更标准的接口。
+这里主要进行修订 Proto 规范约定和多语言之间特定商定，帮助大家写出更标准的接口。
 
-API接口统一以HTTP/GRPC为基础，并通过Protobuf进行协议定义，包括完整的Request/Reply，以及对应的接口错误码（Errors）。
+API 接口统一以 HTTP/GRPC 为基础，并通过 Protobuf 进行协议定义，包括完整的 Request/Reply，以及对应的接口错误码（Errors）。
 
-### 目录结构
-API接口可以定义到项目，或者在统一仓库中管理Proto，类似googleapis、envoy-api、istio-api；
+## 目录结构
+
+API 接口可以定义到项目，或者在统一仓库中管理 Proto，类似 `googleapis`、`envoy-api`、`istio-api`；
+
+项目中定义 Proto，以 api 为包名根目录：
 
-项目中定义Proto，以api为包名根目录：
 ```bash
 kratos-demo：
 |____api // 服务API定义
 | |____kratos
 | | |____demo
-| | | |____v1
+| | | |____v2
```

Comment:
why v2

Reply:
Sorry. I have some error understand. I'm reverted it.